docs/nuget-org/individual-accounts.md

```diff
@@ -31,42 +31,33 @@ To create a NuGet.org account, you need to have a personal Microsoft account (MS
 
    ![Giving permissions to NuGet.org](media/nuget-org-permissions.png)
 
-1. You will be redirected to *nuget.org*, and asked to register a username.
-
-1. Specify the username in the input box. Please note that the username **is** case sensitive and cannot be changed or renamed later.
+1. Follow two-factor authentication (2FA) setup steps, choose either text or e-mail as a second form of authentication.
```

Comment:
Do we want to be specifying any part of the authentication setup? Seems like we should leave it (and any screen shots) to the AAD/MSA docs.

Reply:
Bellow this part on "Add a new individual account", there was documentation for "Enabling two-factor authentication", I basically just copied and pasted that info here, so users know about it. Do you think would be better to only mention "Follow two-factor (2FA) setup steps"? or what links/info do you think would be better to add here?

Reply:
I removed the specific 2FA steps and added those links at the beginning so that users know that they need to have 2fa enabled, and they can do it in advance.